assets/src/edit-story/components/panels/layer/useElements.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+
+import IconBackground from './layer_bg.svg';
+import IconMedia from './layer_media.svg';
+import IconVideo from './layer_video.svg';
+import IconText from './layer_text.svg';
+
+function getIconForElementType( type ) {
+	switch ( type ) {
+		case 'video':
+			return <IconVideo />;
+		case 'text':
+			return <IconText />;
+		case 'image':
+		case 'shape':
+		case 'square': // this is only while square is a "shape"
+			return <IconMedia />;
+		default:
+			return <IconBackground />;
```

Comment:
See above. Perhaps some misunderstanding here. Maybe this should be a distinction between a layer and an element for us and we could have a layer with type "background" instead here?

Reply:
I think I've done a better job explaining this in 07f0e2a.